src/main/java/org/springframework/hateoas/core/JsonPathLinkDiscoverer.java

```diff
@@ -67,23 +67,22 @@
 	}
 
 	private final String pathTemplate;
-	private final MediaType mediaType;
+	private final List<MediaType> mediaTypes;
 
 	/**
 	 * Creates a new {@link JsonPathLinkDiscoverer} using the given path template supporting the given {@link MediaType}.
 	 * The template has to contain a single {@code %s} placeholder which will be replaced by the relation type.
 	 * 
 	 * @param pathTemplate must not be {@literal null} or empty and contain a single placeholder.
-	 * @param mediaType the {@link MediaType} to support.
+	 * @param mediaTypes the {@link MediaType}s to support.
 	 */
-	public JsonPathLinkDiscoverer(String pathTemplate, MediaType mediaType) {
-
+	public JsonPathLinkDiscoverer(String pathTemplate, MediaType... mediaTypes) {
```

Comment:
Wasn't quite sure what to do with this constructor. It seems that the `MediaType` is nullable so I made it varargs. I thought about another constructor with a `List` of `MediaType`...